tests/model_serving/model_runtime/vllm/quantization/test_openhermes-2_5_mistral-7b_awq.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+from simple_logger.logger import get_logger
+from utilities.constants import KServeDeploymentType
+from tests.model_serving.model_runtime.vllm.utils import fetch_openai_response, run_raw_inference
+from tests.model_serving.model_runtime.vllm.constant import VLLM_SUPPORTED_QUANTIZATION
+
+LOGGER = get_logger(name=__name__)
+
+
+serving_argument = [
+    "--model=/mnt/models",
+    "--uvicorn-log-level=debug",
+    "--chat-template=/app/data/template/tool_chat_template_mistral.jinja",
+]
+
+model_path = "TheBloke/OpenHermes-2.5-Mistral-7B-AWQ"
```

Comment:
Shouldn't this be in the constants file?

Reply:
not needed model path is different for each model doesn't make sense to add it to common constant.py